src/main/groovy/org/labkey/gradle/plugin/ModuleResources.groovy

```diff
@@ -66,7 +66,12 @@ class ModuleResources implements Plugin<Project>
                     }
                 }
         if (!LabKeyExtension.isDevMode(project))
-            project.tasks.processModuleResources.doLast(new GzipAction())
+        {
+            GzipAction zipAction = new GzipAction();
+            zipAction.extraExcludes = ["views/**"];
```

Comment:
Or restrict to webapp/** rather than exclude views?

Reply:
I'll look at that.  It requires a bit more fiddling with the GzipAction that's used in a couple of other places, but I could possibly not use it here.